.github/workflows/daily.yml

```diff
@@ -27,27 +28,22 @@ jobs:
         # Defaults for special-case influencing vars
         lein-profile: ['']
         drop-joins: ['']
-        deprecated-query-streaming: ['']
```

Comment:
This is still referenced below. And it's still in main, if it needs to be removed there.